README.md

```diff
@@ -74,7 +74,28 @@ data = yamale.make_data('./data.yaml')
 yamale.validate(schema, data)
 ```
 
-If `data` is valid, nothing will happen. However, if `data` is invalid Yamale will throw a `ValueError` with a message containing all the invalid nodes.
+If `data` is valid, nothing will happen. However, if `data` is invalid Yamale will throw a `YamaleError` with a message containing all the invalid nodes:
+```python
+try:
+    yamale.validate(schema, data)
+    print('Validation success! 👍')
+except ValueError as e:
+    print('Validation failed!\n%s' % str(e))
```

Comment:
Don't forget to change the code in command_line.py to use `str(e)` too. I'm still getting an error when running from the command line. Here's how I run yamale locally if it helps:
```
python3 -m venv ./venv
. ./venv/bin/activate
pip install -e .
yamale -s my.schema my.yaml
```